app/services/PageBuilder.scala

```diff
@@ -130,9 +130,9 @@ class PageBuilder extends ProcessPopulation {
   def fromPageDetails[A](pages: Seq[Page])(f: (String, String, String) => A): List[A] =
```

Comment:
As with the view-external-guidance-frontend code I wonder if this method should be in the test code

Reply:
This is used in the ApprovalService in the save() fn.